app/assets/stylesheets/tpi/_bubble-chart.scss

```diff
@@ -0,0 +1,40 @@
+$tape-height: 8px;
+$tape-color: rgba(25,25,25,0.1);
+
+.bubble-chart__container {
```

Comment:
Sorry for nitpicking but can we use `--element__modifier`instead?

Reply:
yes! sorry, my BEM is rusty

Reply:
done